include/wil/wistd_functional.h

```diff
@@ -393,9 +399,9 @@ namespace wistd     // ("Windows Implementation" std)
         return *this;
     }
 
-    template<class _Rp, class ..._ArgTypes>
+    template<class _Rp, class..._ArgTypes>

```

Comment:
While I don't necessarily mind the `class ..._ArgTypes` syntax (copied from libc++; ideally prefer `class... _ArgTypes`), we should at the very least retain at least one space